main.go

```diff
@@ -29,7 +29,7 @@ import (
 
 var (
 	// Version holds the current Gitea version
-	Version = "1.9.0-dev"
+	Version = "1.12.0-dev"
```

Comment:
I think it would be better if we have this set to 0.0.0-dev, as we have our makefile override this, and then we could also distinguish better when the binary is built via go build without version info passedin.

Reply:
or maybe have it blank like the below variables?